CodingChallenge/MusicSearch/MusicSearchModels.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  MusicSearchModels.swift
+//  CodingChallenge
+//
+//  Created by Edgar Barocio on 6/22/19.
+//  Copyright (c) 2019 Edgar Barocio. All rights reserved.
+//
+//  This file was generated by the Clean Swift Xcode Templates so
+//  you can apply clean architecture to your iOS and Mac projects,
+//  see http://clean-swift.com
+//
+
+import UIKit
+
+enum MusicSearch {
+    // MARK: Use cases
+    
+    enum Search {
+        struct Request {
+            var searchTerm: String
+        }
+        struct Response {
+            var searchResult: [SongResponse]?
+        }
+        struct ViewModel {
+            var searchResult: [SongResponse]?
+        }
```

Comment:
This is kinda cool. When you're rested up, I want you to teach me about this! :D